### ISSUE TITLE
Development roadmap!

### ISSUE THREAD

**Issue by RahulR100** [RahulR100/mcstatusbot#208]
In an effort to promote some transparency on what we are currently working on, I have created a new development roadmap which you can see here: https://github.com/users/RahulR100/projects/2. New issues will also be created on my fork of the repository (https://github.com/RahulR100/mcstatusbot/issues) with additional details for each ticket.

If you want to request a feature, simply open a new issue with the title Feature: <your feature> and Ill take a look at it. Thanks again for all your support as we continue our climb through 33k servers!

**Comment by Leanderrrrr** [comment]
If I want to try develop a feature, can I? Your contribution MD doesn't include guidelines for code additions

**Comment by RahulR100** [comment]
Yes absolutely we would love that! I am currently preparing to push the november update out over thanksgiving. I will also include an update to the contribution.md for code additions with that update.

If you wanna start on something before then, shoot me an email (rahul_rao@oulook.com) with the idea you had and Ill lyk the best way of getting involved!
